go-controller/pkg/ovn/controller/services/node_tracker.go

```diff
@@ -146,7 +148,7 @@ func (nt *nodeTracker) Start(nodeInformer coreinformers.NodeInformer) (cache.Res
 // updateNodeInfo updates the node info cache, and syncs all services
 // if it changed.
 func (nt *nodeTracker) updateNodeInfo(nodeName, switchName, routerName, chassisID string, l3gatewayAddresses,
-	hostAddresses []net.IP, podSubnets []*net.IPNet, zone string, migrated bool) {
+	hostAddresses []net.IP, podSubnets []*net.IPNet, zone string, nodePortDisabled, migrated bool) {
```

Comment:
nit: I think it would be cleaner to not inverse boolean variables. Like in some places its nodePortEnabled and here nodePortDisabled. Could we just use nodePortEnabled everywhere?

Reply:
@trozet I got exactly the same comments from downstream and here is my justification: I inversed it purposefully and all the "make check" unit testsing is assuming nodePortEnabled to true, in order for the unit test to pass, I'll have to identify all the tests and explicitly set nodePortEnabled to true in those test cases.

Reply:
ah ok, then its fine